spec/unit/blockscore/question_set_spec.rb

```diff
@@ -1,42 +1,128 @@
 module BlockScore
   RSpec.describe QuestionSet do
-    let(:person) { create(:person_params) }
+    describe '.new' do
+      subject(:person) do
+        BlockScore::QuestionSet.new
+      end
```

Comment:
I assume this is left over from duplicating the person spec?  Might want to look through this and other files that you bootstrapped with other specs to make sure naming and what not is fine